### PR TITLE
Fix #8190: DefaultCommand: TextEditor contents is not ignored

### LINKED ISSUE #8190: DefaultCommand: TextEditor contents is not ignored
**Describe the defect**
Form with both TextEditor and DefaultCommand makes you not being able to use enter in the editor as it submits the form.

**Reproducer**
```xhtml
    <h:form>
        <h5 class="p-mt-0">Basic</h5>
        <p:textEditor widgetVar="editor1" value="#{editorView.text}" height="300"  />

        <h3 class="first">Custom Toolbar</h3>
        <p:textEditor widgetVar="editor2" value="#{editorView.text2}" height="300" placeholder="Enter your content">
            <f:facet name="toolbar">
             <span class="ql-formats">
                <button class="ql-bold"></button>
                <button class="ql-italic"></button>
                <button class="ql-underline"></button>
                <button class="ql-strike"></button>
            </span>
                <span class="ql-formats">
                <select class="ql-font"></select>
                <select class="ql-size"></select>
            </span>
            </f:facet>
        </p:textEditor>

        <p:commandButton id="fh" value="gj" update="@form"/>
        <p:defaultCommand target="fh"/>
    </h:form>
```

**Environment:**
 - PF Version: 12.0.0-SNAPSHOT
 - JSF + version: Mojarra 2.3
 - Affected browsers: Chrome

### PR DESCRIPTION
**Pull request by jepsar**
Fix #8190

### CODE REVIEW

File: primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.defaultcommand.js [hunk 1]
```diff
@@ -43,9 +43,11 @@ PrimeFaces.widget.DefaultCommand = PrimeFaces.widget.BaseWidget.extend({
...
             data = data || e.data;
             if (($this.scope && data.scopeEnter && data.scopeDefaultCommandId === $this.id)
-                    || (!$this.scope && !data.scopeEnter && (e.which == keyCode.ENTER))) {
-                //do not proceed if target is a textarea,button or link
-                if ($(e.target).is('textarea,button,input[type="submit"],a')) {
+                    || (!$this.scope && !data.scopeEnter && (e.which === keyCode.ENTER))) {
+                // Do not proceed if target is a textarea, button or link
+                // #8190 Do not proceed if target is part of TextEditor content
+                if ($(e.target).is('textarea,button,input[type="submit"],a')
+                        || $(e.target).closest('.ql-editor').length) {
```
Comment (jepsar) [inline primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.defaultcommand.js]: This (line 50) is the actual change

**Review by melloware — APPROVED** [review]

**Comment by tandraschko** [comment]
whats `$(e.target)` in case of the editor?

**Comment by jepsar** [comment]
It is `div.ql-editor`. Do you want me to change the check?

Screenshot 2021-12-28 at 13 37 26

**Comment by tandraschko** [comment]
yep, that would be much faster
just also check ".ql-editor"

**Comment by tandraschko** [comment]
also tested? :D

**Comment by jepsar** [comment]
Yes, it's working like this.

Screenshot 2021-12-28 at 13 52 52

**Pull request merged**